tox.ini

```diff
@@ -0,0 +1,28 @@
+[tox]
+envlist = py26,py27,lint,docs
```

Comment:
Why are we testing against py26 here? Do we really support it? Django 1.8 requires at least Python 2.7 https://docs.djangoproject.com/en/dev/releases/1.8/